examples/golang/add-object.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+)
+
+const (
+	id  = "AWS_ACCESS_KEY_ID"
+	key = "AWS_SECRET_ACCESS_KEY"
+)
+
+func main() {
+	endPoint := flag.String("endpoint", "", "region of request")
+	accessId := flag.String("accessId", "", "Access Id")
+	accessKey := flag.String("accessKey", "", "Access Key")
+	ObjectName := flag.String("objectPath", "", "Object Path")
+	bucketName := flag.String("bucketName", "", "The name of the s3 bucket")
```

Comment:
do you create the bucket in this program?
if not, you should explain in the readme that the bucket has to be created before calling your program.

Reply:
Thank you. Let me get to work immediately.